app/models/video_stream_moderator.py

```diff
@@ -0,0 +1,20 @@
+from app.models import db
+from app.models.base import SoftDeletionModel
+
+
+class VideoStreamModerator(SoftDeletionModel):
```

Comment:
No soft deletion please. It is broken and doesn't work with relationships. It was shoehorned in API without much thinking